src/bin-utils/index.js

```diff
@@ -161,22 +171,41 @@ function requireDefaultFromModule(modulePath) {
   }
 }
 
+function stringifyScript({name, description, script}) {
+  const coloredName = chalk.green(name)
+  const coloredScript = chalk.gray(script)
+  let line
+  if (description) {
+    line = [coloredName, chalk.white(description), coloredScript]
+  } else {
+    line = [coloredName, coloredScript]
+  }
+  return line.join(' - ').trim()
+}
+
+function specificHelpScript({scripts}, scriptName) {
+  const availableScripts = getAvailableScripts(scripts)
+  const script = find(
```

Comment:
Instead of using `find` here, let's use `prefixMatches` from `prefix-matches` (already installed). That way people can still use a prefix to get help on a script.